packages/core/contracts/cross-chain-oracle/chain-adapters/Optimism_ParentMessenger.sol

```diff
@@ -47,7 +47,7 @@ contract Optimism_ParentMessenger is OVM_CrossDomainEnabled, ParentMessengerInte
     function setChildOracleSpoke(address newOracleSpoke) public onlyOwner {
         bytes memory dataSentToChild = abi.encodeWithSignature("setOracleSpoke(address)", newOracleSpoke);
         sendCrossDomainMessage(childMessenger, defaultGasLimit, dataSentToChild);
-        emit MessageSentToChild(dataSentToChild, childMessenger, defaultGasLimit);
+        emit MessageSentToChild(dataSentToChild, childMessenger, defaultGasLimit, childMessenger);
```

Comment:
These events are breaking CI

Reply:
ye, I fixed the same thing on one of my PRs. had the same issue